### ISSUE TITLE
Include the '_' char in the subword

### ISSUE THREAD

**Issue by matheuslessarodrigues** [dsandstrom/atom-subword-navigation#16]
When navigating the subwords, it'd be cool if we don't step into the '_' char as a subword.

For example (after some subword navigations to the right):
|test_word => test_|word => test_word|

Instead of:
|test_word => test|_word => test_|word

I'm sorry I'm new to this opensource world and I'm not able to create a simple pull request :( However, I found that changing the "subwordRegExp" method in the "subword-navigation.coffee" script to this may do the work:

```coffeescript
subwordRegExp: (options={}) ->
    nonWordCharacters = atom.config.get('editor.nonWordCharacters')
    segments = ["^[\t ]*$"]

    if options.backwards
      segments.push("[A-Z]?[a-z]+_?")
      segments.push("_?[A-Z]?[a-z]+")
    else
      segments.push("_?[A-Z]?[a-z]+")
      segments.push("[A-Z]?[a-z]+_?")

    segments.push("[A-Z]+(?![a-z])")
    segments.push("\\d+")

    if options.backwards
      segments.push("[#{_.escapeRegExp(nonWordCharacters)}]+\\s*")
    else
      segments.push("\\s*[#{_.escapeRegExp(nonWordCharacters)}]+")
    new RegExp(segments.join("|"), "g")
```

Thanks!

**Comment by dsandstrom** [comment]
Ah yes, I've been debating whether to change this or not.  How you suggest, is how Sublime does it.  If I get time, I will change it this weekend.  Or make it an option.

**Comment by dsandstrom** [comment]
I added tests: https://github.com/dsandstrom/atom-subword-navigation/tree/include-underscore.  But, your solution wasn't working for me.  I will play around with it more.

**Comment by dsandstrom** [comment]
Got it working.  Let me manually test it out before merging.

**Comment by dsandstrom** [comment]
Fixed by v2.6.0
